release_notes_generator/action_inputs.py

```diff
@@ -219,6 +228,10 @@ def validate_inputs():
         verbose = ActionInputs.get_verbose()
         ActionInputs.validate_input(verbose, bool, "Verbose logging must be a boolean.", errors)
 
+        release_notes_title = ActionInputs.get_release_notes_title()
+        if not isinstance(release_notes_title, str) or len(release_notes_title) == 0:
+            errors.append("Release Notes title must be a non-empty string and have non-zero length.")
```

Comment:
Partially fixed in commit a32f3ca9b1073d16bc325b39e3ece82808e3e81d.

- I do not see reason for this: _Please add a logger for new action inputs release_notes_title as well._

Reply:
In the validate function you are validating the action inputs and at the end of the method you are providing in the debug mode the validated action inputs to the user. I think `release_notes_title` option should be logged there as well, because it detects this new regex feature. It is part of GH action inputs just as `warnings` which is logged.

Reply:
I see. Added in commit a2abdfc30f57e29f2dc666ddc2c43cf2a014a48e.